code/modules/mob/dead/observer/observer.dm

```diff
@@ -1,6 +1,10 @@
 var/global/list/image/ghost_darkness_images = list() //this is a list of images for things ghosts should still be able to see when they toggle darkness
 var/global/list/image/ghost_sightless_images = list() //this is a list of images for things ghosts should still be able to see even without ghost sight
 
+/mob/dead

```

Comment:
Why are these at `/mob/dead` when they only seem to apply to `/mob/dead/observer`?